src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -127,7 +131,13 @@ class StreamGroup extends React.Component<Props, State> {
     }
 
     const data = GroupStore.get(id) as Group;
-    this.setState({data});
+    this.setState(state => {
+      const collapse =
```

Comment:
just making sure I understand the logic correctly, is it:

if the user is on the inbox tab, and there was an inbox reason for the group, but it is now not in the inbox --> then it will be greyed out?

Reply:
oh also is there a reason you call the variable collapse instead of 'reviewed'?

Reply:
Bad search and rename, added a comment. 

You've got it, when it had an inbox reason but no longer does i want to set the state.